src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/DraftOrdersCategoriser.java

```diff
@@ -25,55 +23,35 @@ public DraftOrdersCategoriser() {
         super();
     }
 
-    public void categoriseDocuments(FinremCaseData finremCaseData, String userRole) {
+    public void categoriseDocuments(FinremCaseData finremCaseData) {
         // Determine type of draft order
         if (!isSuggestedDraftOrderPriorToHearing(finremCaseData)) {
             return;
         }
 
-        // Get the current user case role
-        String chosenParty = determineChosenParty(finremCaseData, userRole);
+        UploadSuggestedDraftOrder uploadSuggestedDraftOrder = finremCaseData.getDraftOrdersWrapper()
+            .getUploadSuggestedDraftOrder();
+        OrderFiledBy orderParty = uploadSuggestedDraftOrder.getOrderFiledBy();
 
-        DocumentCategory category = determineCategory(chosenParty);
-        if (category != null) {
-            categoriseOrders(finremCaseData.getDraftOrdersWrapper()
-                .getUploadSuggestedDraftOrder()
-                .getUploadSuggestedDraftOrderCollection(), category);
-
-            categorisePsas(finremCaseData.getDraftOrdersWrapper()
-                .getUploadSuggestedDraftOrder()
-                .getSuggestedPsaCollection(), category);
-        }
+        DocumentCategory category = getDocumentCategory(orderParty);
 
+        categoriseOrders(uploadSuggestedDraftOrder.getUploadSuggestedDraftOrderCollection(), category);
+        categorisePsas(uploadSuggestedDraftOrder.getSuggestedPsaCollection(), category);
     }
```

Comment:
Think we should put JDOC in when it's been omitted previously?

Reply:
Added to categoriseDocuments